docs/whatsnew/3.0.rst

```diff
@@ -38,3 +38,56 @@ Full change log
 
 To see a detailed list of all changes in version v3.0, including changes in
 API, please see the :ref:`changelog`.
+
+
+Renamed/removed functionality
```

Comment:
Nitpick: Should "removed" be capitalized by grammatical rules?

Reply:
No, I think this is fine.

Reply:
If there is a precedent, then it is okay. :)